godtools/ViewControllers/BaseViewController.swift

```diff
@@ -183,4 +184,18 @@ class BaseViewController: UIViewController {
         present(alert, animated: true, completion: nil)
     }
 
+    // MARK: - Analytics Helpers
+    
+    private func sendScreenViewNotification() {
+        let userInfo = [GTConstants.kAnalyticsScreenNameKey: screenName()]
+        
+        NotificationCenter.default.post(name: .screenViewNotification,
+                                        object: nil,
+                                        userInfo: userInfo)
+    }
+    func screenName() -> String {
+//        assertionFailure("method must be overriden by view controller")
+        
+        return "unknown"
```

Comment:
Yes, I wanted to confer with @frett to make sure we have the same naming pattern for our screens on both platforms.